src/model/markercollection.js

```diff
@@ -88,9 +88,10 @@ export default class MarkerCollection {
 	 * @param {String|module:engine/model/markercollection~Marker} markerOrName Name of marker to set or marker instance to update.
 	 * @param {module:engine/model/range~Range} range Marker range.
 	 * @param {Boolean} [managedUsingOperations=false] Specifies whether the marker is managed using operations.
+	 * @param {Boolean} [affectsData=true] Specifies whether the marker changes the data model.
```

Comment:
Marker doesn't change anything itself. Its the presence of the marker that does it. Also, it's not about the data model. All kinds of markers are in the data model. It's about the persistence of markers. Maybe something like this:

> Specifies whether the marker affects the data produced by the data pipeline (is persisted in the editor's data).

This repeats a at least in two more place in this file.